secrets/file.go

```diff
@@ -167,7 +168,7 @@ func addPath(secrets secretMap, path string) {
 		}
 		for _, match := range matches {
 			data, err := readSecretFile(match)
-			if err != nil {
+			if err != nil && !errors.Is(err, syscall.EISDIR) {
```

Comment:
@szuecs 
I've played with https://pkg.go.dev/os#Readlink and https://pkg.go.dev/path/filepath#EvalSymlinks but that becomes a mess quickly so I think it is easier to just mute `Failed to read path /etc/skipper/hostname-credentials/..data: read /etc/skipper/hostname-credentials/..data: is a directory` errors.

See also https://stackoverflow.com/questions/18062026/resolve-symlinks-in-go